src/Plugin/MergePlugin.php

```diff
@@ -146,4 +158,14 @@ private function recognizePluginInstallation(PackageEvent $event): void
             $this->isFirstInstall = true;
         }
     }
+
+    private function getBasePath(RootPackageInterface $rootPackage): string
+    {
+        $basePath = $rootPackage->getExtra()['base_path'] ?? null;
+        if (!empty($basePath) && is_dir($basePath)) {
+            return $basePath;
+        } else {
+            return self::MODULES_BASE_PATH;
+        }
```

Comment:
```suggestion
        if (empty($basePath) || !is_dir($basePath)) {
            return self::MODULES_BASE_PATH;
        }

        return $basePath;
```
und noch ein `use function dir` 🙂